website/docs/basics/blocks-component-in-block.md

```diff
@@ -5,3 +5,75 @@ sidebar_label: Component In Block
 ---
 
 [![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
+
+Blocks and components go perfectly together because they are made this way. We simplified this as much as possible but there always a few things you need to do manually.
+
+Try to think of your development process in this way:
+
+- Each component must be standalone and not depend on and different components (sometimes this is not possible but keep this to a minimum).
+- All components must not be aware of the layout it is used in.
+- You don't add global styles as you did in Bootstrap before.
+- You should define all headings in a heading component, and everywhere you have a heading, use this component and not a new implementation of it. (this is an example, but this implies to every component).
```

Comment:
```suggestion
- You should define all heading variations in a `heading` component. Then use this component everywhere you have a heading instead of writing a new implementation. In practice: don't do `<h2>$heading</h2>` but render the `heading` component instead `Components::render( 'heading', $attributes )` (Heading is an example, but this implies to every component).
```